### PR TITLE
add weight mention, closes #377

### LINKED ISSUE #377: Weighting training examples
@cgreene will make a small pull request in the data limitations sub-section of discussion to add [this](https://github.com/greenelab/deep-review/pull/367#discussion_r114076546) after we merge #367

### PR DESCRIPTION
**Pull request by cgreene**
@agitter : This is my attempt to briefly mention how correction has been done & to highlight that part of the goal of our review is to emphasize these types of techniques. Let me know if this is what you had in mind.

### CODE REVIEW

File: sections/06_discussion.md [hunk 1]
```diff
@@ -80,8 +80,14 @@ training instances can be just as problematic.  In genomics, for example,
 labeled data may be derived from an experimental assay that has known and
 unknown technical artifacts, biases, and error profiles.  It is possible to
 weight training examples or construct Bayesian models to account for uncertainty
-in the data, but this is rarely done in practice. `TODO: @cgreene will edit
-this last line; it might be done in practice but not highlighted in papers`
+or non-independence in the data. For example, Park et al.
+[@doi:10.1371/journal.pcbi.1002957] estimated shared non-biological signal
```
Comment (agitter) [inline sections/06_discussion.md]: Do you think we need to point out this was with a SVM, not deep learning?  I don't want to make the paragraph any longer than necessary though.

File: sections/06_discussion.md [hunk 2]
```diff
@@ -80,8 +80,14 @@ training instances can be just as problematic.  In genomics, for example,
...
+between datasets to correct for non-independence related to assay platform or
+other factors in a Bayesian integration of many datasets. However, such
+techniques are rarely placed front and center in the description of methods, so
+these steps may be overlooked. In this review we have tried to highlight the
```
Comment (agitter) [inline sections/06_discussion.md]: I like the strong sentence "However, such techniques are rarely placed front and center in the description of methods, so these steps may be overlooked."  What do you think about ending the paragraph there and cutting the "In this review...".

Comment (cgreene) [inline sections/06_discussion.md]: I agree. I'll go with that.

File: sections/06_discussion.md (hunk 1 above)
Comment (cgreene) [inline sections/06_discussion.md]: From my read, I think it doesn't matter too much. It's the Bayesian bit of that paper that's corrected for non-independence.

Comment (agitter) [inline sections/06_discussion.md]: That works for me.

Comment (agapow) [inline sections/06_discussion.md]: Agreed, leave as is.

**Pull request merged**